uptane/services/timeserver.py

```diff
@@ -19,7 +19,7 @@
 try:
  import pyasn1.type
 except ImportError:
- print('Minor: pyasn1 library not found. Proceeding using JSON only.')
+ uptane.logger.error('pyasn1 library not found. Proceeding using JSON only.')
 else:
  #import uptane.ber_encoder as ber_encoder
```

Comment:
Outdated code?

Reply:
Why do you ask? (BTW, I haven't really done much testing lately of the behavior when pyasn1 isn't available. It's something I have to figure out.)

Reply:
Sorry, I mean that line 24 is commented out.  It's also an unexpected place to import a module.

Reply:
Ah! Yes, I think this PR is a good place to finally get rid of that, thanks..... (: